CONTRIBUTING.md

```diff
@@ -34,22 +34,22 @@ license document, but changing it is not allowed.
 
 Developer's Certificate of Origin 1.1
 
-By making a contribution to this project, I certify that:
+By contributing to this project, I certify that:
```

Comment:
I think this is directly taken from https://developercertificate.org/

I would not modify text in this code block. It should exactly match it's source.

Reply:
I'm working on it.